src/Reflex/Dom/Widget/Input.hs

```diff
@@ -212,6 +214,110 @@ dropdown k0 options (DropdownConfig setK attrs) = do
   dValue <- combineDyn readKey options =<< holdDyn (Just k0) (leftmost [eChange, fmap Just setK])
   return $ Dropdown dValue (attachDynWith readKey options eChange)
 
+
+data ButtonGroup t a
+   = ButtonGroup { _buttonGroup_value    :: Dynamic t (Maybe a)
+                 , _buttonGroup_select   :: Event  t (Maybe a)
+                 , _buttonGroup_element  :: El t
+                 , _buttonGroup_children :: Dynamic t (Map.Map Int (El t))
+                 }
```

Comment:
In the reflex-dom codebase, the style has been to not vertically align things like the `::` operators here.  Let's stay consistent with that.